### PR TITLE
Fix weird jump after tab animation

### PR DESCRIPTION
**Pull request by yasminyin96**
* Fix weird jump after tab animation

### How was this tested?
- [X] I tested this manually

### Screenshots and screen recordings

https://user-images.githubusercontent.com/104786010/171488931-ef358244-b7f1-4e3f-a42d-a4d67426023b.mov

## Issues addressed
* https://github.com/neevaco/neeva-ios/issues/3734

### CODE REVIEW

File: Client/Frontend/Browser/Card/Card.swift [hunk 1]
```diff
@@ -231,6 +231,7 @@ struct Card<Details>: View where Details: CardDetails {
                                 .padding(1.5)
                                 .padding(tabDetails.isSelected ? 0 : -1.5)
                                 .contextMenu(menuItems: tabDetails.contextMenu)
+                                .padding(tabDetails.isSelected ? -1.5 : 0)
```
Comment (evancrow) [inline Client/Frontend/Browser/Card/Card.swift:234]: Isn't this the exact opposite of the padding above? Does this add the extra padding back to the `ContextMenu`.

Comment (yasminyin96) [reply Client/Frontend/Browser/Card/Card.swift:234]: I think this padding is added to the button.
`
    button
         .if(!animate) { view in
                view
                    .padding
                      ...
`

Comment (yasminyin96) [reply Client/Frontend/Browser/Card/Card.swift:234]: Line 231, it adds padding 1.5 to all tab grids.
Line 232, it changes the padding of unselected tab grid to the original padding in order to remove the unnecessary edge when open the contextMenu.
Line 234, it changes the padding of selected tab to the original padding.

Comment (evancrow) [reply Client/Frontend/Browser/Card/Card.swift:234]: Makes sense, please add a comment though for future reference :)

**Pull request merged**